at_ve_doctor/pubspec.yaml

```diff
@@ -1,13 +1,23 @@
 name: at_ve_doctor
 description: A sample command-line application.
+publish_to: none
 # version: 1.0.0
 # homepage: https://www.example.com
 
 environment:
   sdk: '>=2.8.1 <3.0.0'
 
 dependencies:
-  at_server_status: ^0.1.0
+  at_server_status:
+    git:
+      url: https://github.com/atsign-foundation/at_libraries.git
+      path: at_server_status
+      ref: dev_env_staging
```

Comment:
ref is dev_env or dev_env_staging ? in automated flow, we cannot change after we push to dev_env

Reply:
Sure Murali, I will remove the staging branches